index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Tribute Page</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/style.css">
+    
+  </head>
+
+  <body>
+    <main>
+      <div class="jumbotron header">
+        <h1 id="title" >AYO MOSES ADEYEMI</h1>
+        <p>The Man Who Won A Million Souls To Christ!!!!</p>
+      </div>
```

Comment:
I will also like to suggest that you use a more semantic tag like the header tag to describe this section of your web page. semantic tags like header, nav, etc. are preferred over div's